streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -289,6 +293,11 @@
                     "/tmp/kafka-streams",
                     Importance.HIGH,
                     STATE_DIR_DOC)
+            .define(MIN_IN_SYNC_REPLICAS_CONFIG,
```

Comment:
I don't think priority HIGH is a good choice for this. I guess, using broker default setting is good enough. MEDIUM (ie, "please double check"), might be better IMHO.

Nit: can you pleas add this in alphabetical order (with whatever level we use).